(SIMP-8228) Work around hidepid issues

trevor-vaughan opened the pull request:
- Added the following classes
  - polkit::install
  - polkit::service
  - polkit::user
- Ensure that the polkit user is managed by default and placed into the
  supplementary group bound to the 'gid' option on '/proc' if one is set to work
  around issues with 'hidepid' > 0.

Requires: simp/pupmod-simp-simplib#229

SIMP-8228 #comment Updated polkit

jeannegreulich reviewed (changes requested):
1)  polkit is not included anywhere, so I think we should add it to the defines so it is included.  (I thought it might have been included in simp::classes but it is not)
2) If you do not set the group, it gives you a warning, which is expected, but italso restarts the service every puppet run.  
3) The group is not created anywhere.  Are we going to create it in simp::mountpoints::proc?  I set up a little manifest to create it in site so it would work.  How ever it took 2 puppet runs for polkit to add polkitd to the group because the group was not there the first run.  So we would need a relationship between the 2 or a note in polkits that it could take 2 runs to set up.

trevor-vaughan commented:
@jeannegreulich I think I have everything resolved:
* Added the includes to the defines
* Added an acceptance test to ensure that items are idempotent
* The group will be created via `simp::mountpionts::proc`. The current implementation here just uses whatever group is assigned which allows for proper separation of intent and state
